Obsidian/Server.cs

```diff
@@ -265,31 +268,31 @@ public async Task RunAsync()
         await WorldManager.LoadWorldsAsync();
 
         if (!Config.OnlineMode)
-            Logger.LogInformation($"Starting in offline mode...");
+            _logger.LogInformation($"Starting in offline mode...");
 
         Registry.RegisterCommands(this);
 
         if (Configuration.EnableRcon)
         {
-            rconServer = new RconServer(LoggerProvider.CreateLogger("RCON"), Config, this, CommandsHandler);
+            rconServer = new RconServer(_logger, Config, this, CommandsHandler);
```

Comment:
You're adding `RconServer` as a service through `services.AddObsidian` as well... doesn't this produce two Rcon servers?

Reply:
Nice catch! Will fix.